### PR TITLE
add documentation - phase 1

### PR DESCRIPTION
**Pull request by pg2455**
Adding documentation to some functions that I recently touched.

### CODE REVIEW

**Comment by plstcharles** [comment]
sweet sweet documentation, me gusta --- be aware however that some of these changes will conflict with the open PRs on experimental/develop, and those PRs should be merged ASAP to avoid having to rewrite everything again

File: src/covid19sim/base.py [hunk 1]
```diff
@@ -1118,14 +1128,14 @@ def update_book(self, human, date=None, risk_level = None):
...
-        [summary]
+        Sends messages to all `Human`s in self.book.
 
         Args:
-            owner ([type]): [description]
-            tracing_method ([type]): [description]
-            order (int, optional): [description]. Defaults to 1.
-            reason (str, optional): [description]. Defaults to "test".
-            payload ([type], optional): [description]. Defaults to None.
+            owner ([Human]): owner of this contact book.
```
Comment (vict0rsch) [inline src/covid19sim/base.py]: I'm sorry to be this annoying person @pg2455 but type docs should be

```
owner (Human): owner of this contact book.
```

For reference, here's a complete example: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

Comment (pg2455) [reply src/covid19sim/base.py]: so annoying ... 💢 👎

Comment (pg2455) [reply src/covid19sim/base.py]: done 👍

**Review by plstcharles — APPROVED** [review]
because prateek prefers green to red

**Review by vict0rsch — APPROVED** [review]

**Pull request merged**